docs/contents/dev-docs/setup.md

```diff
@@ -11,21 +11,95 @@ the steps below.
     between different contributors and should in general help keeping everyone
     happy.
 
-## Fork the upstream repository
+## Prerequisites
 
-First, fork the [upstream source repository](https://github.com/aws/aws-controllers-k8s) 
-into your personal GitHub account. Then, in `$GOPATH/src/github.com/aws/`,
-clone your repo and add the upstream like so:
+Please ensure that you have [properly installed Go][install-go].
 
-```
-git clone git@github.com:$GITHUB_ID/aws-controllers-k8s && \
-cd aws-controllers-k8s && \
-git remote add upstream git@github.com:aws/aws-controllers-k8s
-```
+[install-go]: https://golang.org/doc/install
 
 !!! note "Go version"
     We recommend to use a Go version of `1.14` or above for development.
 
+## Fork upstream repositories
+
+The first step in setting up your ACK development environment is to fork the
+upstream ACK source code repositories to your personal Github account.
+
+There are two common upstream repositories you should fork first:
+
+* `github.com/aws/aws-controllers-k8s` is the core ACK runtime and types
+* `github.com/aws-controllers-k8s/code-generator` is the ACK code generator
+
+!!! tip "prefix ACK forked repos with 'ack-'"
+    When I fork repositories to my personal Github account, I tend to prefix
+    the repositories with a common string for grouping purposes. For ACK source
+    repositories that I forked from the `github.com/aws-controllers-k8s` Github
+    Organization, I prefix those repositories with "ack-". For example, when I
+    orked the `github.com/aws-controllers-k8s/code-generator` repository to my
```

Comment:
I've "orked" a few repos in my past but I don't think that's the word you wanted to use here.

Reply:
:) fixed.